webapi/src/main/scala/org/knora/webapi/routing/admin/lists/OldListsRouteADMFeature.scala

```diff
@@ -34,7 +34,7 @@ object OldListsRouteADMFeature {
 }
 
 /**
- * A [[Feature]] that provides the old list admin API route.
+ * Provides the old list admin API route.
```

Comment:
There is no new route, so old one should be removed too. Maybe the name of the class `OldListsRouteADMFeature`, which is also related to feature toggle, could be updated too?

Reply:
Good idea, we should make it a habit to clean-up things while we do other stuff. I refactored the admin lists route. Let's see if the tests still pass.